account_payment_mode/models/account_payment_mode.py

```diff
@@ -0,0 +1,118 @@
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class AccountPaymentMode(models.Model):
+    """This corresponds to the object payment.mode of v8 with some
+    important changes. It also replaces the object payment.method
+    of the module sale_payment_method of OCA/e-commerce"""
+    _name = "account.payment.mode"
+    _description = 'Payment Modes'
+    _order = 'name'
+
+    name = fields.Char(required=True, translate=True)
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True, ondelete='restrict',
+        default=lambda self: self.env['res.company']._company_default_get(
```

Comment:
`company_default_get()` is deprecated in V13.

Use:

```python
default=lambda self: self.env.company
```

Reply:
Done!